src/overmap.cpp

```diff
@@ -410,6 +410,8 @@ void oter_type_t::load( JsonObject &jo, const std::string &src )
 
     assign( jo, "sym", sym, strict );
     assign( jo, "name", name, strict );
+    // Store localized name
+    name = _( name.c_str() );
```

Comment:
This would be better:
```c++
if( assign( jo, "name", name, strict ) ) {
    name = _( name.c_str() );
}
```
In the future, we probably should introduce an overloaded `assign()` that accepts some kind of `reader` (like `mandatory()` and `optional()` do).